maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gitexe/src/main/java/org/apache/maven/scm/provider/git/gitexe/command/changelog/GitChangeLogCommand.java

```diff
@@ -89,17 +97,28 @@ protected ChangeLogScmResult executeChangeLogCommand( ChangeLogScmRequest reques
         final String datePattern = request.getDatePattern();
         return executeChangeLogCommand( request.getScmRepository().getProviderRepository(), fileSet,
             request.getStartDate(), request.getEndDate(), request.getScmBranch(), datePattern, startVersion,
-                endVersion, request.getLimit() );
+                endVersion, request.getLimit(), request.getRevision() );
```

Comment:
Move both to variables for consistency.

Reply:
which both? there are 10 parameters, 4 provided as variables, 6 as method calls on request variable.

Reply:
Let's leave it as-is right now.